index.js

```diff
@@ -27,3 +31,154 @@ for (let i = 0; i < mdlLks.length; i += 1) {
+  button2.classList.add('project-detail__button');
+  // Append
+  container.append(modalHeader, tagsDiv, imgDiv, textDiv);
+  imgDiv.appendChild(mainImg);
+  textDiv.append(leftDiv, rigthDiv);
+  leftDiv.appendChild(description);
+  rigthDiv.append(techDiv, line, buttonsDiv);
+  modalHeader.append(h2, exitButton);
+  tagsDiv.append(tag1, bullet1, tag2, bullet2, tag3);
+  buttonsDiv.append(button1, button2);
+  buttonsDiv.classList.add('project-detail__buttons');
+  button1.innerText = 'See live';
+  button1.appendChild(btnImg1);
+  button2.innerText = 'See source';
+  button2.appendChild(btnImg2);
+}
+
+// To iterate elements in portfolio data
+for (let i = 0; i < data.length; i++) {
+  // Creating elements
+  const portDiv = document.createElement('div');
+  portDiv.classList.add('port1');
+  const portImg = document.createElement('img');
+  const rghtDiv = document.createElement('div');
+  const portTtl1 = document.createElement('h2');
+  portTtl1.classList.add('title');
+  const deetsDiv = document.createElement('div');
+  const span1 = document.createElement('span');
+  span1.classList.add('clnt');
+  const span2 = document.createElement('span');
+  span2.classList.add('deets');
+  const bullet = document.createElement('img');
+  const bullet2 = document.createElement('img');
+  bullet.classList.add('crcl');
+  bullet2.classList.add('crcl');
+  const span3 = document.createElement('span');
+  const description = document.createElement('p');
+  const tchnlgs = document.createElement('ul');
+  tchnlgs.classList.add('tags');
+  // Technologies for loop
+  for (let j = 0; j < data[i].technologies[j]; j++) {
+    const li = document.createElement('li');
+    li.classList.add('tgs');
+    li.innerText = data[i].tags[j];
+    tchnlgs.appendChild(li);
+  }
+  const btnDiv = document.createElement('div');
+  const btn = document.createElement('button');
+  btn.classList.add('prjctbtn');
+  const btnTxt = document.createElement('span');
+  btnTxt.classList.add('btntxt');
+  // Adding content
+  portImg.src = data[i].image;
+  portImg.alt = `${data[i].title} img`;
+  portTtl1.innerText = data[i].title;
+  span1.innerText = data[i].tags[0];
+  span2.innerText = data[i].tags[1];
+  span3.innerText = data[i].tags[2];
+  bullet.src = 'images/circleicn.png';
+  bullet2.src = 'images/circleicn.png';
+  description.innerText = data[i].description;
+  // Append
+  workSectn.appendChild(portDiv);
+  portDiv.appendChild(portImg);
+  portDiv.appendChild(rghtDiv);
+  rghtDiv.appendChild(portTtl1);
+  rghtDiv.appendChild(deetsDiv);
+  deetsDiv.appendChild(span1);
+  deetsDiv.appendChild(bullet);
+  deetsDiv.appendChild(span2);
+  deetsDiv.appendChild(bullet2);
+  deetsDiv.appendChild(span3);
+  rghtDiv.appendChild(description);
+  rghtDiv.appendChild(tchnlgs);
+  portDiv.appendChild(btnDiv);
+  btnDiv.appendChild(btn);
+  btnTxt.innerText = 'See Project';
+  btn.appendChild(btnTxt);
+  btn.addEventListener('click', () => {
+    projectDetails(data[i]);
+  });
+}
```

Comment:
**OPTIONAL** - For this project, you were not required to render your project cards dynamically, You are only required to render the popup modal dynamically in javascript. I suggest that you return your project cards back into the HTML file. Please fix this 🙏 
- As a result of the changes you made, your project card section does not look like how it is in the design template, please note that I am making this change required because you only introduced this change in this milestone. Please fix this 🙏 
Your project (now):
<img width="937" alt="rgq3wgqg" src="https://user-images.githubusercontent.com/73607512/216138535-02cf01f5-e4a0-40bc-8f64-f9f8fef07f3a.PNG">
Your project(before now):
<img width="943" alt="gq3gq3g3" src="https://user-images.githubusercontent.com/73607512/216138640-ff154431-ef75-4857-9567-9a502ba76bb0.PNG">